Makefile

```diff
@@ -32,7 +32,7 @@ build-all:
 
 .PHONY: build-node ## Build release node with `tee-dev` feature
 build-node:
-	cargo build --locked -p $(call pkgid, $(NODE_BIN)) --release --features=tee-dev
+	~/.cargo/bin/cargo build --locked -p $(call pkgid, $(NODE_BIN)) --release --features=tee-dev --package=litentry-collator
```

Comment:
I don't think this change is necessary:
`-p $(call pkgid, $(NODE_BIN))` should point to `litentry-collator` already